packages/gestalt/src/behaviors/TrapFocusBehavior.js

```diff
@@ -29,7 +29,7 @@ function queryFocusableAll(el: HTMLDivElement) {
 
 const focusElement = (el: HTMLElement) => {
   // https://github.com/facebook/flow/issues/8705
-  // $FlowFixMe[method-unbinding]
+  // $FlowIssue[method-unbinding]
```

Comment:
Unrelated to the rest of the changes — using a more precise suppression here to indicate that it's an issue on Flow's side